example/src/main/java/org/geysermc/mcprotocollib/network/example/PingPacket.java

```diff
@@ -3,19 +3,15 @@
 import org.geysermc.mcprotocollib.network.codec.PacketCodecHelper;
 import org.geysermc.mcprotocollib.network.packet.Packet;
 import io.netty.buffer.ByteBuf;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
 
+@Getter
+@RequiredArgsConstructor
```

Comment:
i don't want to use lombok for the example - i would just revert all the changes to this class